src/malli/core.cljc

```diff
@@ -1,7 +1,7 @@
 (ns malli.core
   (:refer-clojure :exclude [eval type -deref deref -lookup -key])
   #?(:cljs (:require-macros malli.core))
-  (:require #?(:clj [clojure.walk :as walk])
+  (:require [clojure.walk :as walk]
```

Comment:
I'm not sure why clojure.walk was only required in clj mode. Perhaps to reduce cljs bundle size? If so, I can move the data-explainer to malli.util, or alternatively rewrite it without using clojure.walk.

Reply:
good point, moved to malli.util